inspirehep/records/indexer/tasks.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import logging
+
+from celery import shared_task
+from elasticsearch import NotFoundError
+from invenio_db import db
+from invenio_pidstore.models import PersistentIdentifier
+from sqlalchemy import tuple_
+from sqlalchemy.orm.exc import NoResultFound, StaleDataError
+
+from inspirehep.records.api import InspireRecord
+from inspirehep.records.errors import MissingArgumentError, MissingCitedRecordError
+from inspirehep.records.indexer.base import InspireRecordIndexer
+
+logger = logging.getLogger(__name__)
+
+
+def get_record(uuid, record_version=None):
```

Comment:
why we keep this here? can we just move the logic inside the InspireRecord?

Reply:
This function is used only by tasks. If any other part of the code will start using this, then we can do this, but for now adding another method inside InspireRecord which already looks messy is overkill and over complication in my opinion.